.github/labeler.yml

```diff
@@ -1,46 +1,53 @@
-# Add 'Multicast' label to any root file changes
-Multicast:
-  - setup.py
-  - multicast/*.py
+# Add 'Python Repo' label to any root file changes
+Python Repo:
```

Comment:
It is unclear if the whitespace here needs quoted from the docs:
 - see https://github.com/actions/labeler/blob/506e1a00c6c05ba98ac585056a23f16073fa5915/README.md for details